.composer.json

```diff
@@ -36,18 +36,20 @@
     "test:behavioral": [
       "@test:behat-cli -c Neos.ContentRepository.BehavioralTests/Tests/Behavior/behat.yml.dist",
       "@test:behat-cli -c Neos.ContentGraph.DoctrineDbalAdapter/Tests/Behavior/behat.yml.dist",
-      "../../flow doctrine:migrate --quiet; ../../flow cr:setup",
-      "@test:behat-cli -c Neos.Neos/Tests/Behavior/behat.yml",
       "@test:behat-cli -c Neos.ContentRepository.LegacyNodeMigration/Tests/Behavior/behat.yml.dist",
-      "@test:behat-cli -c Neos.TimeableNodeVisibility/Tests/Behavior/behat.yml.dist"
+      "@test:behat-cli -c Neos.ContentRepository.Export/Tests/Behavior/behat.yml.dist",
+      "@test:behat-cli -c Neos.TimeableNodeVisibility/Tests/Behavior/behat.yml.dist",
+      "../../flow doctrine:migrate --quiet; ../../flow cr:setup",
```

Comment:
This puzzles me!?

Reply:
Thanks for keeping things tidy. Its different than you think ;)
I just moved the `"../../flow doctrine:migrate --quiet; ../../flow cr:setup` part two lines below as its only seemingly required for the Neos.Neos tests and not the rest.
We should definitely get rid of this hack and its only needed in the ci but idk why;)

Reply:
yeah and believe me im also confused about the fact its in the default development context ... seems to fix a glitch will take care of this at one point ;)

Reply:
you're right, this wasn't introduced with your PR but much [earlier](https://github.com/mhsdesign/neos-development-collection/commit/3fd279dd6727e0433ff5de4346710eb55ea981ed)
But..
```
./flow cr:setup
```
in development context has nothing to do with the behat context – or only by accident.

Are you sure that this fixes a glitch, or did we just drag this along?

Reply:
@mhsdesign lets see what the CI says: #5005